galaxy_ng/app/models/container.py

```diff
@@ -1,10 +1,15 @@
 from django.db import models
 from pulpcore.plugin import models as pulp_models
-from pulp_container.app import models as container_models
+# from pulp_container.app import models as container_models
 from django_lifecycle import LifecycleModelMixin
 
 from galaxy_ng.app.access_control import mixins
 
+try:
+    from pulp_container.app import models as container_models
```

Comment:
this seems pretty extreme. Why is it here? If we can't import the container models, then our models are going to be unusable.

Reply:
https://issues.redhat.com/browse/AAH-2386